frontend/src/components/StatusTransfer.tsx

```diff
@@ -0,0 +1,126 @@
+
+      const {chain: {network}} = useNetwork()
+
+    const handleAddress = address => {
+        const sub_domain = network === 'sepolia' ? 'sepolia.' : ''
+        const url = `https://${sub_domain}starkscan.co/contract/${getAddressToHex(address)}`
+        window.open(`${url}`, '_blank')
+    }
+
+    const renderData = () => {
+        if (Array.isArray(data)) {
+            if (data.length === 0) {
+                return <div className='p-2'>There are no transfers</div>;
+            }
+
+            return data.map((transfer_item, index) => {
+
+                return (
+                    <div key={index} className="flex flex-wrap flex-row">
+                        <div className='flex flex-wrap flex-row w-full'>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider border-r border-slate-400">token</div>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider">receiver</div>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 overflow-hidden cursor-pointer" onClick={() => handleAddress(transfer_item.token_addr)}>{getFormatAddress(transfer_item.token_addr.toString())}</div>
+                            <div className="flex-1 w-1/2 basis-1/2 p-2 overflow-hidden cursor-pointer" onClick={() => handleAddress(transfer_item.receiver)}>{getFormatAddress(transfer_item.receiver.toString())}</div>
+                        </div>
+                        <div className='flex flex-wrap flex-row w-full'>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider">amount</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider">cooldown_end</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider">status</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 bg-slate-200 text-left text-gray-700 uppercase tracking-wider">actions</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 ">{formatBalance(transfer_item.amount.toString())}</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 ">{getHoursLeft(transfer_item.cooldown_end.toString())}</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 ">{getTransferStatus(transfer_item.status)}</div>
+                            <div className="flex-1 w-1/4 basis-1/4 p-2 ">
+                                { renderCancelBtn(transfer_item?.status, transfer_item?.id) }
+                            </div>
+                        </div>
+
+                    </div>
+                )
+            })
+        } else {
+            return null
+        }
+    }
+
+
+    useEffect(() => {
```

Comment:
What is the purpose of the refetch() here? It’s not needed elsewhere.

Reply:
without refetch get_live_transfers does not make a request

Reply:
if you run refetch without useEffect is constant requests and re-render

Reply:
@Oleh-Zorin I just pushed a version without the refetch. Issue was missing `args: [],` in useContractRead.

My original question was – why do we need refetch *in this component and not elsewhere*. The answer is, args are needed and the refetch provides the (empty) args.